example/lib/develop/peripheral_list/peripheral_list_screen.dart

```diff
@@ -0,0 +1,129 @@
+import 'dart:io';
+
+import 'package:blemulator_example/develop/model/ble_peripheral.dart';
+import 'package:blemulator_example/develop/peripheral_list/bloc.dart';
+import 'package:blemulator_example/develop/peripheral_list/components/connection_state_view.dart';
+import 'package:blemulator_example/develop/peripheral_list/components/rssi_view.dart';
+import 'package:blemulator_example/develop/util/custom_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class PeripheralListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return _buildTestLayout(context);
+  }
+
+  Widget _buildTestLayout(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Bluetooth peripherals'),
+        // TODO: - replace with a toggle button based on state.scanningEnabled
+        actions: <Widget>[
```

Comment:
should both buttons be enabled all the time? maybe a user should not be able to press `stop if there is not ongoing scanning?

Reply:
Totally agree with that, this one is marked as TODO and it's going to be addressed.

Reply:
Fixed.